packages/presets/src/__tests__/edgeless/frame.spec.ts

```diff
@@ -93,14 +97,16 @@ describe('frame', () => {
     )!;
     const bodyRect = frameBody.getBoundingClientRect();
 
+    console.log(titleRect);
```

Comment:
console log

Reply:
This test has some issues on CI; this line of code is used to debug the test on CI.

Reply:
Done!